sdk/confidentialledger/azure-confidentialledger/tests/test_identity_service_client.py

```diff
@@ -1,38 +1,25 @@
-import sys
+from devtools_testutils import AzureTestCase
 
-from azure.confidentialledger.identity_service import (
+from azure.confidentialledger_identity_service import (
```

Comment:
double clicking on this title, is this the approved final name? it doesn't feel very pythonic to me

Reply:
Who do I talk to about this? I wasn't sure if `azure.identityservice` would be ok, while nesting the modules (as it used to be i.e. `azure.confidentialledger.identity_service`) resulted in DPG clearing out the nested module when the outer module was regenerated.

Reply:
Will ask Johan for input